.github/CODEOWNERS

```diff
@@ -771,6 +771,12 @@
 # ServiceLabel: %Image Analysis
 # ServiceOwners:                                                   @rhurey @dargilco
 
+# PRLabel: %Planetary Computer
+/sdk/planetarycomputer/                                            @chahibi @shaktichetan23
```

Comment:
Both of the folks on this line need to have their `Microsoft` membership marked as public to be valid code owners.